public/app/features/plugins/sandbox/sandbox_plugin_loader.ts

```diff
@@ -72,6 +72,11 @@ async function doImportPluginModuleInSandbox(meta: PluginMeta): Promise<System.M
       // distortions are interceptors to modify the behavior of objects when
       // the code inside the sandbox tries to access them
       distortionCallback,
+      defaultPolicy: {
```

Comment:
Did you manage to import the global default policy here?

Reply:
I created an issue for this https://github.com/grafana/grafana/issues/75290 I decided I don't want this to PR to include potentially breaking functionality. I am not sure how trusted types behave inside the sandbox and I'll have to do more testing before I introduce our actual policies.

There's also the possibility that our trusted types policies are not necessary at all in the sandbox since we have many many other protection mechanisms.